src/main/java/com/adyen/model/payout/SubmitRequest.java

```diff
@@ -0,0 +1,231 @@
+/*
+ *                       ######
+ *                       ######
+ * ############    ####( ######  #####. ######  ############   ############
+ * #############  #####( ######  #####. ######  #############  #############
+ *        ######  #####( ######  #####. ######  #####  ######  #####  ######
+ * ###### ######  #####( ######  #####. ######  #####  #####   #####  ######
+ * ###### ######  #####( ######  #####. ######  #####          #####  ######
+ * #############  #############  #############  #############  #####  ######
+ *  ############   ############  #############   ############  #####  ######
+ *                                      ######
+ *                               #############
+ *                               ############
+ *
+ * Adyen Java API Library
+ *
+ * Copyright (c) 2017 Adyen B.V.
+ * This file is open source and available under the MIT license.
+ * See the LICENSE file for more info.
+ */
+package com.adyen.model.payout;
+
+import java.util.Date;
+import java.util.Map;
+import com.adyen.model.Amount;
+import com.adyen.model.Name;
+import com.adyen.model.recurring.Recurring;
+import com.adyen.serializer.DateSerializer;
+import com.google.gson.annotations.JsonAdapter;
+import com.google.gson.annotations.SerializedName;
+
+public class SubmitRequest {
+    @SerializedName("additionalData")
+    private Map<String, String> additionalData = null;
+
+    @SerializedName("amount")
+    private Amount amount = null;
+
+    @SerializedName("dateOfBirth")
+    @JsonAdapter(DateSerializer.class)
+    private Date dateOfBirth = null;
+
+    @SerializedName("entityType")
+    private String entityType = null;
+
+    @SerializedName("fraudOffset")
+    private Integer fraudOffset = null;
+
+    @SerializedName("merchantAccount")
+    private String merchantAccount = null;
+
+    @SerializedName("selectedRecurringDetailReference")
+    private String selectedRecurringDetailReference = null;
+
+    @SerializedName("nationality")
+    private String nationality = null;
+
+    @SerializedName("recurring")
+    private Recurring recurring = null;
+
+    @SerializedName("shopperEmail")
+    private String shopperEmail = null;
+
+    @SerializedName("reference")
+    private String reference = null;
+
+    @SerializedName("shopperName")
+    private Name shopperName = null;
+
+    @SerializedName("shopperReference")
+    private String shopperReference = null;
+
+    @SerializedName("shopperStatement")
+    private String shopperStatement = null;
+
+    @SerializedName("socialSecurityNumber")
+    private String socialSecurityNumber = null;
+
```

Comment:
Can you please add the constructor with: 

Recurring recurring = new Recurring();
recurring.setContract(Recurring.ContractEnum.PAYOUT);
this.recurring = recurring;

Like you did in the other files?